app/views/group_offers/show.html.slim

```diff
@@ -68,23 +68,19 @@ h1= @group_offer.title
             td= t_attr(:schedule_details)
             td= @group_offer.schedule_details
 
-- if !@group_offer.external? && @group_offer.responsible
-  - if @group_offer.volunteer_responsible_state?
-    h3= t("volunteer_responsible_state.#{@group_offer.volunteer_responsible_state}")
-  - else
-    h3= t_attr(:responsible)
-  .row
-    .col-xs-12
-      ul
-        li= link_to @group_offer.responsible, volunteer_path(@group_offer.responsible)
-
-- if @group_offer.volunteers.any?
+- if @group_offer.group_assignments.any?
   h3= t_attr(:volunteers)
-  .row
-    .col-xs-12
-      ul
-        - @group_offer.volunteers.each do |volunteer|
-          li= link_to volunteer, volunteer_path(volunteer)
+  ul
+    - @group_offer.group_assignments.each do |group_assignment|
+      li
+        .col-xs-12.col-md-3
+          = link_to group_assignment.volunteer, volunteer_path(group_assignment.volunteer)
+        .col-xs-12.col-md-3
+          = group_assignment.responsible ? t_attr(:responsible, GroupAssignment) : t_attr(:member, GroupAssignment)
+        .col-xs-12.col-md-3
+          = l(group_assignment.period_start) if group_assignment.period_start
+        .col-xs-12.col-md-3
+          = l(group_assignment.period_end) if group_assignment.period_end
 
 - unless params[:format] == 'pdf'
```

Comment:
<img width="780" alt="screen shot 2017-10-13 at 13 52 08" src="https://user-images.githubusercontent.com/11731325/31544990-bc259d38-b01d-11e7-9b2c-56e420259bf8.png">

the pdf needs some styling for the volunteers list